README.md

```diff
@@ -1,75 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio: Setup mobil first
 
 > Description the project.
 
+**For this repository, I created html templates and styling files in order to built a page github of Microverse project.**
+
+![alt text](https://github.com/Carl2678/setup-mobile-first-part-2/blob/main/images/Template%201-Mobile-Main.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Git & Github
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://github.com/CarlytoDev/CarlytoDev.github.io)
```

Comment:
The live demo link in your readme is not working, it just direct you to the github repository. Kindly check and ensure that it is working properly. :wink: